Assets/VRTK/SDK/Daydream/DAYDREAM_README.md

```diff
@@ -0,0 +1,194 @@
+	* Add GvrControllerMain prefab from Assets/GoogleVR/Prefabs/Controller/
+	* Add GvrViewerMain prefab (enables view in editor play mode)
+* Disable Daydream's native pointer tools
+	* To the Camera object, disable or remove GvrPointerPhysicsRaycaster component, if present
+	* To the GvrControllerPointer/Laser, disable or delete
+
+
+### Setup VRTK Components
+* In Hierarchy, create an Empty named "[VRTK]"
+* Add component VRTK_SDKManager
+* Add a child Empty named "RightController"
+* Note, Daydream supports only one controller, LeftController will not be used. If present, can be disabled or deleted.
+* SDK Selection
+	* In Inspector, choose Quick Select SDK: Daydream
+	* that should populate the four SDKs, 
+	* In Player Settings, ensure Scripting Define Symbols: VRTK_SDK_DAYDREAM
+* Linked Objects: 
+	* Click "Auto Populate Linked Objects", that should set:
+	* Actual Boundaries: DaydreamCameraRig
+	* Actual Headset: DaydreamCameraRig/Camera
+	* Actual Left Controller: empty
+	* Actual Right Controller: DaydreamCameraRig/GvrControllerPointer/Controller
+* Controler Aliases:
+	* Model Alias Left Controller: empty
+	* Model Alias Right Controller: DaydreamCameraRig/GvrControllerPoints/Controller
+	* Script Alias Left Controller: empty
+	* Script Alias Right Controller: [VRTK]/RightController
+
+
+## Tips
+
+* Note, we're using the name "RightController" to follow VRTK convention. However, it's a Daydream device setting whether the controller is rendered as held in the player's right or left hand. But normally players will have just one controller.
+
+* I recommend enabling Use Accelerometer in GrvControllerMain for more natural hand tracking.
+
+* You can change the Controller's GvrArmModelOffsets Joint to Elbow for more exaggerated (amplified) motions
+
+* Optionally, in Hierarchy add the following prefabs,
+	* GvrFPSCanvas (to see the frames per second rate)
+	* DemoInputManager (to see the controller connection status)
+
+
+### VRTK Controller Components
+Depending on which components you're using on the [VRTK]/RightController, you might want to make the following adjustments.
+
+* VRTK_ControllerEvents: suggested Action Alias Buttons:
+	* Pointer Toggle Button: Touchpad_Touch
+	* Pointer Set Button: Touchpad_Touch
+	* Grab Toggle Button: Touchpad_Press
+	* Use Toggle Button: Touchpad_Press
+	* UI Click Button: Touchpad_Press
+	* Menu Toggle Button: Button_One_Press
+
+* VRTK_SimplePointer: 
+	* PointOriginTransform: drag in the GvrControllerPointer/Laser object, which has the Point joint (instead of Controller which has the Wrist joint) or other object using Pointer joint 
+	* (Note: Pointer Tilt Angle is set in GvrControllerMain's GvrArmModel component)
+
+### VRTK Event Support
+
+The following shows the events supported directly by the VRTK Daydream SDK. Other events, such as Trigger_Press, if needed should be set in the controller Action Alias Buttons.
+
+**Supported**
+* Touchpad_Touch
+* Touchpad_Press
+* Button_One_Press
```

Comment:
I just mean that you may want to use the `Button_Two` as your default button press rather than `Button_One` as the Vive wand doesn't support a `Button One` but supports a `Button Two` (in VRTK aliases)